### PR TITLE
add unit tests and CI integration

### PR DESCRIPTION
**Pull request by kevinluvian**
https://bugzilla.mozilla.org/show_bug.cgi?id=1456571

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: python
+  python:
+    - "3.6"
+  script:
+    - pytest
```
Comment (MihaiTabara) [inline .travis.yml:5]: This looks good. Nit, not blocking: as a follow-up we can introduce the use of `tox` to ease the running of these both locally and in Travis.

File: tests/test_shipitapi.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+def fake(*args, **kwargs):
+	kwargs.pop('sleeptime')
+	kwargs.pop('max_sleeptime')
+	return real_retry(sleeptime=1, max_sleeptime=1, jitter=1, *args, **kwargs)
+
+
+redo.retry = fake
+
+
+def test_simple():
```
Comment (MihaiTabara) [inline tests/test_shipitapi.py:22]: I guess we can remove this test, I suppose it was for testing purposes, right?

File: tests/test_shipitapi.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	"""
+	csrf token format is %Y%m%d%H%M%S##...
+	"""
+	return expiry.strftime('%Y%m%d%H%M%S') + '##....'
+
+
+def test_csrf_token_expired(mocker):
+	from src.shipitapi.shipitapi import is_csrf_token_expired
+
+	now = datetime.datetime.utcnow()
```
Comment (MihaiTabara) [inline tests/test_shipitapi.py:36]: This looks good, but just FYI - playing around with time can also be done via the `freezegun` lib. There's an example on how to use this in [this](https://github.com/mozilla-releng/shipitscript/blob/master/shipitscript/test/integration/test_integration_script.py#L49) file.

**Review by MihaiTabara — APPROVED** [review]
This looks good, thanks again for your contribution. There's some tiny nits which can be done in a follow-up PR so I don't think it's worth blocking this. I'll merge this to see if we can get the Travis unblocked;

Quick obs: we can rename the `tests` folder to `test`, to be consistent with the other repos

File: .travis.yml (hunk 1 above)
Comment (kevinluvian) [reply .travis.yml:5]: I am not really familiar with tox, how do I get started on that?

File: tests/test_shipitapi.py (hunk 2 above)
Comment (kevinluvian) [reply tests/test_shipitapi.py:22]: I previously thought that this test would help any runtime issue, but I guess you are correct, we can use any test to do it. I'll remove it!

File: tests/test_shipitapi.py (hunk 3 above)
Comment (kevinluvian) [reply tests/test_shipitapi.py:36]: I see! I'll just change it to use freezegun, since it looks cleaner, thanks!

**Comment by kevinluvian** [comment]
@MihaiTabara updates for your comments is on this PR https://github.com/mozilla-releng/shipitapi/pull/8

File: .travis.yml (hunk 1 above)
Comment (MihaiTabara) [reply .travis.yml:5]: Commented more about this in your follow-up PR https://github.com/mozilla-releng/shipitapi/pull/8.

**Comment by MihaiTabara** [comment]
Made more comments there as well. Thanks for following-up.

**Pull request merged**